Slim/Schema/ResultSet/Work.pm

```diff
@@ -0,0 +1,9 @@
+package Slim::Schema::ResultSet::Work;
+
+
+use strict;
+use base qw(Slim::Schema::ResultSet::Base);
+
+use Slim::Utils::Prefs;
```

Comment:
not needed

Reply:
You mean the whole thing? Or just the prefs import like Composer.pm?

I suppose my wider question is do we need these ResultSet scripts when there's nothing in them? This is where my knowledge fails me.

Reply:
Some code might try to use it, like eg. `objForUrl` or similar - where you specifically exclude it... TBH: I don't know whether they're required or not. If you don't know where it's being used, then it likely wouldn't even work in case it was used. I believe the base class has some code to provide search or lookup. But I don't know whether it's generic enough to cover your use case.

If you don't know whether it's being used at all, you could remove it. And we'd add it once we figured out where it was missing?